assignment1.js

```diff
@@ -0,0 +1,69 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+
+
+function secondLargest(array) {
+ var i; var sh;
+var fh=array[0];
+if(array.length > 2) {
+          for(i=0;i<array.length;i++) {
```

Comment:
What will you do If I ask you to get me the 5th largest, will you write the loop five times?